cmd/get/task_util.go

```diff
@@ -68,7 +58,7 @@ func FetchTaskLatestVersion(ctx context.Context, name string, project string, do
 	var err error
 	// Fetch the latest version of the task.
 	var taskVersions []*admin.Task
-	taskVersions, err = FetchAllVerOfTask(ctx, name, project, domain, cmdCtx)
+	taskVersions, err = FetchAllVerOfTask(ctx, name, cmdCtx)
```

Comment:
Lets keep the old way of passing project and domain if possible to avoid config dependency in every module.
That way we can try to restrict this config dependency everywhere and also it becomes harder to test this way.